app/components/CategoryOverviewView.js

```diff
@@ -3,10 +3,12 @@ import PropTypes from 'prop-types';
 import { View } from 'react-native';
 
 import CategoryHeaderAndroid from './CategoryHeader.android';
+import ProductList from '../containers/ProductList';
```

Comment:
Please avoid such hierarchy (pulling containers from other's container's component - view).
We should move all `ProductList` mapping to `CategoryOverview` and pass props downstream.